packages/lwc/__tests__/default-exports.spec.ts

```diff
@@ -20,6 +21,7 @@ const expectExportDefaultFromPackageInFile = (pkgName: string, ext: string) => {
 };
 
 /*
+ * This comment needs to be updated:
  * Jest uses CommonJS, which means that packages with no explicit export statements actually export
  * the default `module.exports` empty object. That export is an empty object with the prototype set
  * to an empty object with null prototype.
```

Comment:
This needs to be reviewed: Is hasExplicitDefaultExport still needed?

Reply:
Now that we use Vitest, it looks like we can change this to simply `expect(aliasedModule.default).toBe(realModule.default)`. But that can wait for a future PR.